builder/builder.go

```diff
@@ -1760,6 +1785,29 @@ func (b *Builder) BuildImage(format string, template string) error {
 	return imagecmd.Run()
 }
 
+// Updates bundle list in the default image template file
+func updateImageTemplate(filePath string, bundles []string) error {
+	f, err := os.Open(filePath)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	byteValue, _ := ioutil.ReadAll(f)
+	var data map[string]interface{}
+	if err := json.Unmarshal([]byte(byteValue), &data); err != nil {
+		return err
+	}
+
+	data["Bundles"] = bundles
+	b, err := json.MarshalIndent(data, " ", " ")
```

Comment:
Why do you supply a `prefix`? That looks weird in the result:
```go
package main

import (
	"encoding/json"
	"fmt"
	"log"
)

func main() {
	data := map[string]int{
		"a": 1,
		"b": 2,
	}

	json, err := json.MarshalIndent(data, "<prefix>", "<indent>")
	if err != nil {
		log.Fatal(err)
	}

	fmt.Println(string(json))
}
```
produces
```
{
<prefix><indent>"a": 1,
<prefix><indent>"b": 2
<prefix>}
```

Reply:
Fixed. Empty prefix.